cli/cmd/compose/up.go

```diff
@@ -73,6 +77,12 @@ func upCommand(p *projectOptions, contextType string) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			switch contextType {
 			case store.LocalContextType, store.DefaultContextType, store.EcsLocalSimulationContextType:
+				if opts.exitCodeFrom != "" {
```

Comment:
Nit: we could remove the switch case here and validate opts regardless of context type